backend/src/config/db.config.ts

```diff
@@ -27,14 +25,16 @@ export const connect = () => {
       acquire: 20000,
       idle: 5000,
     },
+    models: [User],
```

Comment:
would be better if we use the `getAllModelsInDir` function instead of loading them manually